### PR TITLE
Clarify that issues must be created before PRs

### PR DESCRIPTION
**Pull request by tigrannajaryan**
The contributing guide says all significant changes should go through OTEP
process. It also describes an issue workflow which says that an issue is
created first, the issue is triaged and accepted and then a PR is created,
however the "Proposing a change" section does not explicitly describe this
as a possibility.

I added a paragraph to make it clear that there are 3 possible options:
with an OTEP, with an issue and trivial changes that are just PRs.

Unless we make this change it appears that everything that is not a trivial
change must go through OTEP, which is not our current process and is not
what we want.

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,8 +16,18 @@ Agreement](https://identity.linuxfoundation.org/projects/cncf).
 Significant changes should go through the [OpenTelemetry Enhancement
 Proposal](https://github.com/open-telemetry/oteps) process.
 
-Small changes, such as clarifications, wording changes, spelling/grammar
-corrections, etc. can be made directly via pull requests.
+Smaller changes can follow a shorter process:
+
+- Create an issue in this repo and describe the proposed change.
```
Comment (reyang) [inline CONTRIBUTING.md:21]:
```suggestion
- [Create an issue](https://github.com/open-telemetry/opentelemetry-specification/issues/new/choose) in this repo and describe the proposed change.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,8 +16,18 @@ Agreement](https://identity.linuxfoundation.org/projects/cncf).
...
+- Follow the issue workflow and make sure the issue is accepted with a "Yes" response.
```
Comment (reyang) [inline CONTRIBUTING.md:22]: Accepted/yes from whom?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,8 +16,18 @@ Agreement](https://identity.linuxfoundation.org/projects/cncf).
...
+  If the response to the issue is not "Yes" then do not create a PR that implements
+  the change since it may be rejected.
```
Comment (reyang) [inline CONTRIBUTING.md:24]: "may be rejected" - sounds like there is still a chance to get the PR accepted?

File: CONTRIBUTING.md (hunk 2 above)
Comment (reyang) [reply CONTRIBUTING.md:22]: Consider adding a link to the issue workflow.

File: CONTRIBUTING.md (hunk 1 above)
Comment (beeme1mr) [reply CONTRIBUTING.md:21]: The current available templates are all geared towards issues and vulnerabilities. Please consider adding an additional template for proposals.

**Review by carlosalberto — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (tigrannajaryan) [reply CONTRIBUTING.md]: Changed to "will".

File: CONTRIBUTING.md (hunk 2 above)
Comment (tigrannajaryan) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tigrannajaryan) [reply CONTRIBUTING.md]: Sounds good. Added the link for now.

**Review by reyang — APPROVED** [review]

**Pull request merged**